frontend/src/components/widgets/buttons/TextButton.svelte

```diff
@@ -22,7 +22,7 @@
 	class:emphasized
 	class:disabled
 	class:sharp-right-corners={sharpRightCorners}
-	style:min-width={minWidth > 0 ? `${minWidth}px` : undefined}
```

Comment:
That's too bad there's a warning from `undefined` :( In general I try to forcibly use `undefined` everywhere to standardize on it instead of `null`, even so some JS APIs which return `null` get often converted to `undefined` with the `||` operator where that's safe. But if this produces warnings, I guess we gotta stick with `null`.

Reply:
I did some research and found that an empty string works instead of `null`, which helps us avoid `null` throughout our JS codebase in favor of `undefined`.